libs/community/tests/unit_tests/chat_models/test_baichuan.py

```diff
@@ -85,65 +84,34 @@ def test__convert_delta_to_message_human() -> None:
     assert result == expected_output
 
 
-def test__signature() -> None:
-    secret_key = SecretStr("YOUR_SECRET_KEY")
-
-    result = _signature(
-        secret_key=secret_key,
-        payload={
-            "model": "Baichuan2-53B",
-            "messages": [{"role": "user", "content": "Hi"}],
-        },
-        timestamp=1697734335,
-    )
-
-    # The signature was generated by the demo provided by Baichuan.
-    # https://platform.baichuan-ai.com/docs/api#4
-    expected_output = "24a50b2db1648e25a244c67c5ab57d3f"
-    assert result == expected_output
-
-
 def test_baichuan_key_masked_when_passed_from_env(
     monkeypatch: MonkeyPatch, capsys: CaptureFixture
 ) -> None:
     """Test initialization with an API key provided via an env variable"""
     monkeypatch.setenv("BAICHUAN_API_KEY", "test-api-key")
-    monkeypatch.setenv("BAICHUAN_SECRET_KEY", "test-secret-key")
 
     chat = ChatBaichuan()
     print(chat.baichuan_api_key, end="")
     captured = capsys.readouterr()
     assert captured.out == "**********"
 
-    print(chat.baichuan_secret_key, end="")
-    captured = capsys.readouterr()
-    assert captured.out == "**********"
-
 
 def test_baichuan_key_masked_when_passed_via_constructor(
     capsys: CaptureFixture,
 ) -> None:
     """Test initialization with an API key provided via the initializer"""
     chat = ChatBaichuan(
-        baichuan_api_key="test-api-key", baichuan_secret_key="test-secret-key"
+        baichuan_api_key="test-api-key"
```

Comment:
My suggestion is not to remove this parameter to test compatibility

Reply:
Sure.